ansible/roles/nginx/defaults/main.yml

```diff
@@ -8,13 +8,19 @@ nginx_docker_container: nginx
 nginx_workdir: "/docker/nginx"
 nginx_virtualhosts_workdir: "{{ nginx_workdir }}/conf"
 
-virtualhost:
-  fqdn: example.com
+instances:
```

Comment:
We should find a better name for `instance`. They are not instances by themselves.